src/main/java/use_case/list/removeMedia/RemoveMediaInput.java

```diff
@@ -0,0 +1,25 @@
+package use_case.list.removeMedia;
+
+/**
+ * The Input Data for the remove media Use Case.
+ */
+public class RemoveMediaInput {
+    private String collectionName;
+    private String mediaName;
+
+    public String getCollectionName() {
+        return collectionName;
+    }
+
+    public void setCollectionName(String collectionName) {
+        this.collectionName = collectionName;
+    }
+
+    public String getMediaName() {
+        return mediaName;
+    }
+
+    public void setMovieToRemove(String mName) {
+        this.mediaName = mName;
```

Comment:
Our application is more general than movies, so it'd make more sense to call it setMediaToRemove, just like your getMediaName method.

```suggestion
    public void setMediaToRemove(String mediaName) {
        this.mediaName = mediaName;
```